### DISCUSSION TITLE
Swedish Translation

### DISCUSSION THREAD

**Discussion by Mafrans** [ChristopherKlay/StadiaEnhanced#11] [Translations]
LANG: sv

"default": "Standard",
"native": "Inbyggd",
"hide": "Göm",
"show": "Visa",
"visible": "Synligt",
"hidden": "Gömt",
"enabled": "Aktiverat"
"disabled": "Inaktiverat",
"auto": "Automatisk", 
"manual": "Manuell",
"games": "Spel",
"bundles": "Spel-paket",
"addons": "Tillägg",
"responsive": "Responsiv",
"windowed": "Fönsterläge",
"fullscreen": "Fullskärmsläge",
"searchstore": "Sök i butiken",
"onsale": "På Rea",
"prodeals": "Pro Deals",
"allgames": "Alla Spel",
"usermedia": "Skärmdumpar & Filmer",
"searchbtnbase": "Sök på",
"avatarpopup": "Nytt avatar-URL (lämna tomt för standard):",
"searchheader": "Spel inklusive",
"sessiontime": "Sessionstid",
"codec": "Kodec",
"resolution": "Upplösning",
"hardware": "Hårdvara",
"software": "Mjukvara",
"trafficsession": "Sessionstrafik",
"trafficcurrent": "Nuvarande trafik",
"trafficaverage": "Genomsnittlig trafik",
"packetloss": "Tappade paket",
"framedrop": "Tappade bilder",
"latency": "Latens",
"jitter": "Jitter Buffer",
"decodetime": "Avkodningstid"
"compression": "Kompression",
"streammon": "Strömmonitor"
"stream": "Ström",
"community": "Gemenskap",
"speedtest": "Hastighetstest",
"quickaccess": "Snabbmeny",
"messages": "Meddelanden",
"avatar": "Avatar",
"interface": "Gränssnitt",
"shortcut": "Genvägar",
"shortcuttitle": "Installera en genväg för",
"shortcutdesc": "Låter dig installera en genväg för ett spel på din enhet",
"gridsize": "Rutnätsstorlek",
"griddesc": "Ändrar hur många spel visas per rad på hemskärmen",
"clock": "Klocka",
"clockdesc": "Visar den nuvarande tiden på vänlistan, som en spel-overlay eller båda."
"friendslist":"Vänner",
"igoverlay":"Spelöverlägg",
"listoverlay":"Lista & Överlägg",
"filter": "Filter",
"filterdesc": "Låter dig sortera hemskärmen genom att gömma spel. Filtret kan växlas med symbolen uppe till höver ovanför dina spel på hemskärmen."
"filtertoggle":"Växla Filter",
"filterquick":"Snabbfilter",
"invitebase":"Kopiera inbjudningslänk",
"inviteactive":"Kopierat!",
"prolabel": "Pro-Etikett",
"prolabeldesc": "Gömmer 'Pro'-etiketten från spel på hemskärmen.",
"homegallery": "Användargalleri",
"homegallerydesc": "Gömmer 'Captures'-sektionen längst ner på hemskärmen.",
"quickprev": "Förhandsvisning av Meddelande",
"quickprevdesc": "Gömmer förhandsvisningen av meddelanden i vänlistan.",
"quickrep": "Snabbsvar",
"quickrepdesc": "Gömmer snabbsvarsfunktionen i vänlistan",
"offlinefriend": "Offline-vänner",
"offlinefrienddesc": "Gömmer offline-vänner i vänlistan",
"invisiblefriend": "Osynliga Vänner",
"invisiblefrienddesc": "Gömmer vänner med okänd online-status i vänlistan",
"streammode": "Streamer-läge",
"streammodedesc": "Aktivera för att göra vissa delar (bl.a vänlistan) oläsbar medan du streamar (genom verktyg som OBS / Discord)",
"catprev": "Kategori-förhandsvisning",
"catprevdesc": "Gömmer kategori-taggarna när du har muspekaren över ett spel.",
"popup": "Popup-Effekt",
"popupdesc": "Tar bort inzoomingseffekten när du har muspekaren över ett spel på hemskärmen.",
"streammondesc": "Aktivera för att starta en monitor så fort du öppnar ett spel.",
"resolutiondesc": "Målupplösningen för spel. 1440p och 2160p kräver VP9.",
"codecdesc": "Det kodec som används för spel.",
"confirmreset": "Är du säker på att du vill återställa inställningarna?",
"gamesfinished": "Färdiga Spel",
"achievementsunlocked": "Prestationer Uppnådda",
"splitstore": "Dela Butikslistor",
"splitstoredesc": "Delar butikslistor i två kolumner för en bättre överblick.",
"scrollbar": "Scrollistor",
"scrollbardesc": "Aktiverar scrollistor på hemskärmen och i butiken.",
"resetsettings": "Återställ Inställningar"

**Comment by ChristopherKlay** [comment]
Just updated the translation file with the missing line; `"resolution": "Resolution",`
Would be lovely if you could translate that, so i can push a hotfix for the streaming monitor (:

**Comment by Mafrans** [comment]
Done

**Comment by ChristopherKlay** [comment]
New lines for v1.6.0 and the new settings menu/options.

    "stream": "Stream",
    "community": "Community",
    "speedtest": "Speedtest",
    "quickaccess": "Quick Access",
    "messages": "Messages",
    "prolabel": "Pro Label",
    "homegallery": "User Gallery",
    "quickprev": "Message Preview",
    "quickrep": "Quick Reply",
    "offlinefriend": "Offline Friends",
    "invisiblefriend": "Invisible Friends",
    "avatar": "Avatar"

**Comment by ChristopherKlay** [comment]
New lines for v1.7.0 and the shortcut integration.
```
"shortcutshover":"Autohide Shortcut Buttons",
"shortcutstoggle":"Shortcut Buttons"
```

**Comment by Mafrans** [comment]
Updated with the new translations.
I assumed "Quick Access" was in the context of a button for quick menu access, and have therefore translated it to lit. "Quick Menu". Notify me if that's an issue.

**Comment by ChristopherKlay** [comment]
"Quick Access" is used for header of the menu category that holds shortcuts to the speedtest/community page and others.

Thanks for the update (:

**Comment by ChristopherKlay** [comment]
v1.7.0 was slightly rushed due to changes on Googles end, thus i applied the new version and forgot two strings for 1.7.0;
```
"streammode": "Streaming Mode",
"interface": "Interface"
```
These will be added in a later hotfix when available.

Thank you once more (:

**Comment by Mafrans** [comment]
Updated.
Translated "Streaming Mode" to lit. "Streamer-mode"

**Comment by ChristopherKlay** [comment]
Small update of the translation file for `1.8.4` [here](https://github.com/ChristopherKlay/StadiaEnhanced/blob/master/translations.md#new-content).
No stress however; i'll update translation when available (:

**Comment by Mafrans** [comment]
Updated main document, added:

```
"games": "Spel"
"bundles": "Spel-paket"
"addons": "Tillägg"
"hardware": "Hårdvara"
"software": "Mjukvara"
"decodetime": "Avkodningstid"
```

**Comment by ChristopherKlay** [comment]
Note the changes made in `1.8.0` for the menu massively changed the translation file, as seen [here](https://github.com/ChristopherKlay/StadiaEnhanced/blob/master/translations.md#current-template---updated-10012021).

**Comment by Mafrans** [comment]
Updated, everything should be proper now

**Reply by ChristopherKlay** [reply]
Dang, that was fast.
Thank you (:
